docs/sql-migration-guide.md

```diff
@@ -30,6 +30,8 @@ license: |
 
   - In Spark 3.2, `ALTER TABLE .. RENAME TO PARTITION` throws `PartitionAlreadyExistsException` instead of `AnalysisException` for tables from Hive external when the target partition already exists.
 
+  - In Spark 3.2, script transform default `FIELD DELIMIT` is `\u0001` for no serde mode. In Spark 3.1 or earlier, the default `FIELD DELIMIT` is `\t`.
```

Comment:
Do we need backquote? Maybe ``` `FIELD DELIMIT` ``` -> `FIELD DELIMIT`?

Reply:
> Do we need backquote? Maybe `` `FIELD DELIMIT` `` -> `FIELD DELIMIT`?

Done